<launch>

    <!-- ACTIVAR e iniciar MoveIt para controlar brazo -->
    <!-- <include file="$(find disinfection_process)/launch/COMPLETO_activar_brazo.launch"/> -->

    <!-- Se inicia el servidor del servicio encargado de mover el brazo -->
    <group ns="robot">
        <node type="test_spray.py" name="disinfection_arm_service_server" pkg="disinfection_process" output="screen"/>
    </group>


</launch>